txpool/txpool.go

```diff
@@ -29,6 +29,7 @@ const (
 // errors
 var (
 	ErrIntrinsicGas        = errors.New("intrinsic gas too low")
+	ErrBlockLimitExceeded  = errors.New("transaction's gas limit exceeds block gas limit")
```

Comment:
Please make sure the error corresponds to the same one GETH uses:
https://github.com/ethereum/go-ethereum/blob/f32feeb260054d7c90c1efd57dfec6b717717e18/core/error.go#L60

This is required since many web3 frameworks hardcode these expected values in source code